ProtocolHandler.pm

```diff
@@ -36,7 +52,165 @@ sub new {
 	
 	${*$sock}{contentType} = $mime;
```

Comment:
Please fix whitespace and indentation: use tabs, not spaces. Everywhere.

Reply:
I'll have a look at this, but is quite strange, sure I never wanted to add spaces.